shared_types/src/standings.rs

```diff
@@ -43,7 +43,7 @@ impl Standings {
             players: HashSet::new(),
             players_scores: HashMap::new(),
             pairings: HashMap::new(),
-            tiebreakers: vec![],
+            tiebreakers: vec![Tiebreaker::RawPoints],
```

Comment:
I think this might lead to problems, as the fronted sends a vec of tiebreakers and when the torunament is created that vec will get added on to this one and we end up with RawPoints in there twice.

It probably doesn't make sense to have a tournament where the first tiebreaker isn't points but this would also prevent that.